Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:16.04
```

Comment:
The whole `Dockerfile` can be simplified a lot.

If you use `https://hub.docker.com/_/postgres` (latest image, 13.2) you don't need to install anything - moreover, it already comes with the `postgres` user (so you don't have to change the tests)

Reply:
I know there are images ready to use. This was an exercise to create from scratch and think about a multipurpose image (maybe test multiple kinds of database?) I created another user as a good practice, but will switch back to postgres. Do you want to use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all connections without a password?

Reply:
I guess it's better to depend upon ready-to-use images, each of them specialized (e.g. the mariadb, postgres, mongo, ecc). In this way, the amount of LOC to maintain (for the CI - that's a tool and not the main software) remains low and we can be almost sure that the behavior of the official images is the correct one (instead, creating images by ourself with a melting pot of stuff inside, there's the risk to introduce some unwanted behavior).


> Do you want to use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all connections without a password?

That's a good question.
The PostgreSql service in bacup should be generic hence we should be able to test both cases, with and without a password.

Perhaps a user with password authentication (IIRC is just the md5 configuration in the `hba.conf`, but I can be very wrong) and another user without - so we can test both cases.

Reply:
Ok, I agree. I'll switch to the postgres:latest image.